### DISCUSSION TITLE
Shoutcast Server Address

### DISCUSSION THREAD

**Discussion by ismaelazanza** [AzuraCast/AzuraCast#3996] [Q&A]
Greetings, I would like to ask a question, I have installed Shoutcast in Azuracast, but I see in the configuration that this "LocalHost" my idea is to change for the public IP address, because no link works from the Shoutcast configuration for example I want to listen in M3U and It gives me the link (http: // localhost: 8000 / listen.m3u? sid = 1), I want to know how you can change from localhost to the server's IP address and if this change would not affect the streaming operation. I appreciate your responses.
<img width="305" alt="108583883-ce1ff480-730a-11eb-8be1-51402633e988" src="https://user-images.githubusercontent.com/50606824/113593297-e66a7b00-95fb-11eb-915f-34489aa36298.png">

**Comment by Vaalyn** [comment]
There is most likely some setting for the SHOUTcast DNAS 2 that is currently not present in the automatically generated config.

You can manually add settings to the SC config via the the `Station Profile` -> `Broadcast` -> `Advanced Custom Configuration`.

If you can help me figure out which setting we'd need to add to our generated config I can add this to AzuraCast.
